src-util/modulescript/src/com/etendoerp/copilot/modulescript/CopilotSetup.java

```diff
@@ -0,0 +1,24 @@
+package com.etendoerp.copilot.modulescript;
+
+import org.openbravo.modulescript.ModuleScript;
+
+
+
+import java.sql.PreparedStatement;
+
+
+import org.openbravo.database.ConnectionProvider;
+
+
+public class CopilotSetup extends ModuleScript {
```

Comment:
There is another module script with the same name in https://github.com/etendosoftware/com.etendoerp.copilot/pull/33/files

Reply:
Yeah, its contemplated